smart-contracts/TODO

```diff
@@ -0,0 +1,63 @@
+TODOs
```

Comment:
I think this file can be removed now that we're done?
If there are more things to do, maybe worth opening an issue/ticket and keeping track of stuff there?

Reply:
yes, deleted